### PR TITLE
PipelineRun template variables

### PR DESCRIPTION
**Pull request by MartinBasti**
Documentation about supported template variables by osbs-client

CLOUDBLD-9475

Signed-off-by: Martin Basti <mbasti@redhat.com>

### CODE REVIEW

File: docs/admins.rst [hunk 1]
```diff
@@ -272,6 +272,78 @@ The minimal configuration for binary and source build would include::
...
+   * - Variable
+     - Description
+   * - $osbs_configmap
+     - Name of configmap to be used in build (taken from osbs-client config)
+   * - $osbs_namespace
+     - OSBS namespace used for build (taken from osbs-client config)
+   * - $osbs_pipeline_run_name
+     - Pipeline run name created by OSBS (required)
+   * - $osbs_user_params_json
+     - OSBS user parameters encoded in JSON. it's JSON string don't forget to use iit as `'$osbs_user_params_json'`
```
Comment (chmeliik) [inline docs/admins.rst:344]:
```suggestion
     - OSBS user parameters encoded in JSON. it's JSON string don't forget to use it as `'$osbs_user_params_json'`
```

File: docs/admins.rst [hunk 2]
```diff
@@ -272,6 +272,78 @@ The minimal configuration for binary and source build would include::
   # token = ...
 
 
+
+Pipeline run template
+'''''''''''''''''''''
+Osbs-client requires path to tekton PipelineRun template (``pipeline_run_path``)
+which is used to create PipelineRun tekton object. PipelineRun template must be
+created according OSBS tekton Pipeline definitions with modifications suitable for
```
Comment (chmeliik) [inline docs/admins.rst:280]:
```suggestion
created according to OSBS tekton Pipeline definitions with modifications suitable for
```

**Review by chmeliik — COMMENTED** [review]
Fancy table, first time seeing that in RST :tada:

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

File: docs/admins.rst [hunk 3]
```diff
@@ -272,6 +272,78 @@ The minimal configuration for binary and source build would include::
...
+   * - Variable
+     - Description
+   * - $osbs_configmap
+     - Name of configmap to be used in build (taken from osbs-client config)
+   * - $osbs_namespace
+     - OSBS namespace used for build (taken from osbs-client config)
+   * - $osbs_pipeline_run_name
+     - Pipeline run name created by OSBS (required)
+   * - $osbs_user_params_json
+     - OSBS user parameters encoded in JSON. it's JSON string don't forget to use it as `'$osbs_user_params_json'`
```
Comment (ben-alkov) [inline docs/admins.rst:344]:
```suggestion
     - OSBS user parameters encoded in JSON. It's JSON string; don't forget to use it as `'$osbs_user_params_json'`
```

**Review by rcerven — APPROVED** [review]

**Pull request merged**